From 65a2b31fa89ff27251ae30ad3bb3a22d4ef6dff0 Mon Sep 17 00:00:00 2001
From: Yuval Adam <yuval@y3xz.com>
Date: Mon, 15 Feb 2016 23:08:00 +0200
Subject: [PATCH] Add additional syslinux paths

This path exists on Arch Linux systems, and since liveusb-creator is now
packaged for Arch we should probably add it
---
 tails_installer/creator.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tails_installer/creator.py b/tails_installer/creator.py
index 38ec961..4683c3c 100755
--- a/tails_installer/creator.py
+++ b/tails_installer/creator.py
@@ -997,6 +997,7 @@ class LinuxTailsInstallerCreator(TailsInstallerCreator):
             for com32mod in com32modules:
                 copied = False
                 for orig_dir in ('/usr/lib/syslinux/modules/bios',
+                                 '/usr/lib/syslinux/bios',
                                  '/usr/share/syslinux',
                                  '/usr/lib/syslinux'):
                     com32path = os.path.join(orig_dir, com32mod)
@@ -1227,6 +1228,7 @@ class LinuxTailsInstallerCreator(TailsInstallerCreator):
             for mbr_bin in ('/usr/lib/syslinux/mbr/gptmbr.bin',
                             '/usr/lib/SYSLINUX/gptmbr.bin',
                             '/usr/lib/syslinux/gptmbr.bin',
+                            '/usr/lib/syslinux/bios/gptmbr.bin',
                             '/usr/share/syslinux/gptmbr.bin'):
                 if os.path.exists(mbr_bin):
                     return mbr_bin
-- 
2.7.1

